submit-web/src/components/Projects/ProjectTable/StaffTableRow.tsx

```diff
@@ -25,6 +25,10 @@ export default function StaffTableRow({ submissionPackage }: ProjectRowProps) {
     });
   };
 
+  if (submissionPackage.meta === null) {
```

Comment:
Please pull from develop. meta was changed to json. Also instead of if (something === null) -> if(!something)